src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx

```diff
@@ -11,52 +11,55 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 type CSVFileImportProps = {
-  url: string,
-  title: string
+  url: string;
+  title: string;
 };
 
-export default function CSVFileImport({url, title}: CSVFileImportProps) {
+export default function CSVFileImport({ url, title }: CSVFileImportProps) {
   const classes = useStyles();
   const [file, setFile] = useState<any>();
 
   const onFileChange = (e: any) => {
     console.log(e);
-    let files = e.target.files || e.dataTransfer.files
-    if (!files.length) return
+    let files = e.target.files || e.dataTransfer.files;
+    if (!files.length) return;
     setFile(files.item(0));
   };
 
   const removeFile = () => {
-    setFile('');
+    setFile("");
   };
 
   const uploadFile = async (e: any) => {
-      // Get the presigned URL
-      const response = await axios({
-        method: 'GET',
-        url,
-        params: {
-          name: encodeURIComponent(file.name)
-        }
-      })
-      console.log('File to upload: ', file.name)
-      console.log('Uploading to: ', response.data)
-      const result = await fetch(response.data, {
-        method: 'PUT',
-        body: file
-      })
-      console.log('Result: ', result)
-      setFile('');
-    }
-  ;
-
+    // Get the presigned URL
+    const response = await axios({
+      method: "GET",
+      url,
+      params: {
+        name: encodeURIComponent(file.name),
+        contentType: encodeURIComponent(file.type),
```

Comment:
Why do you use content-type here? The name should be enough:
![image](https://user-images.githubusercontent.com/33042066/99431541-ff5be380-291b-11eb-8001-7d9f983d962e.png)